travharv/web_discovery.py

```diff
@@ -90,32 +91,38 @@ def get_description_into_graph(
     session.mount("http://", adapter)
     session.mount("https://", adapter)
 
-    formats = ["text/turtle", "application/ld+json"]
+    triples_found = False
+
     ACCEPTABLE_MIMETYPES = {
         "application/ld+json",
         "text/turtle",
         "application/json",
     }
-    triples_found = False
 
     for format in formats:
         headers = {"Accept": format}
         log.debug(f"requesting {subject_url} with {headers=}")
         r = session.get(subject_url, headers=headers)
         mime_type, options = cgi.parse_header(r.headers["Content-Type"])
-
         if r.status_code == 200 and bool(mime_type in ACCEPTABLE_MIMETYPES):
             triples_found = True
             try:
-                graph.parse(data=r.text, format=format, publicID=subject_url)
-                log.info(
-                    f"content of{subject_url} added to triplestore in{format=}"
+                # if mimetype is application/json use application/ls+json
```

Comment:
typo in comment ld+json (not ls)

while at it; 
- replace "use" with "assume"
- rephrase follow up sentence with "to satisfy the known formats of rdflib.parser"